src/components/Gallery.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import { GalleryPhotoDetails } from "../api/nasaApi";
+import "./Gallery.scss";
+import "../styles/_variables.scss";
+import Modal from "./Modal";
+interface GalleryProps {
```

Comment:
This is personal preference, but I'd prefer a gap here
```suggestion
import Modal from "./Modal";

interface GalleryProps {
```